### ISSUE TITLE
[Spring] - Server Models - Missing Setter Methods

### ISSUE THREAD

**Issue by apissdks** [swagger-api/swagger-codegen#3793]
When you have a model spec that contains readOnly properties,

```
"Acme":{
   "type":"object",
   "required":[
      "acmeName"
   ],
   "properties":{
      "acmeId":{
         "type":"string",
         "description":"Acme unique identifier",
         "readOnly":true
      },
      "acmeStatus":{
         "type":"string",
         "description":"Acme status",
         "readOnly":true
      },
      "acmeName":{
         "type":"string",
         "description":"Acme name"
      }
   },
   "description":"Acme model"
}
```

The generated server models do not contain the setter methods.

**Comment by rajeshkamal** [comment]
Hi @wing328 any inputs on this. Looks like because of this one - https://github.com/swagger-api/swagger-codegen/pull/2206

Thanks for your time.

**Comment by wing328** [comment]
@rajeshkamal I'll take a look later this week (if you don't see any reply/PR from me by next Monday, please ping me again)

**Comment by wing328** [comment]
(and my apologies for missing this issue)

**Comment by rajeshkamal** [comment]
No issues @wing328 I will ping if I don't see any movement here.

**Comment by apissdks** [comment]
Any updates @wing328

**Comment by wing328** [comment]
@apissdks fixed via #4112. Please pull the latest master to give it another try.
